boards/FSM/Inc/Io/Io_Adc.h

```diff
@@ -13,3 +13,9 @@ uint16_t *Io_Adc_GetRawAdcValues(void);
  * @return The voltage measured at ADC channel 1, in volts
  */
 float Io_Adc_GetChannel1Voltage(void);
+
+/**
+ * Get the voltage measured at ADC channel 3
+ * @return The voltage measured at ADC channel 3, in volts
+ */
+float Io_Adc_GetChannel3Voltage(void);
```

Comment:
@thekenu - As a thought, we can give these channels more descriptive names in CubeMX right? I think it's out-of-scope for this PR, but worth having a discussion about.

Reply:
We actually can _not_ in CubeMX. We can give pin names descriptive names, but no the ADC channel itself. If that was a possibility I would definitely go for it!

Reply:
RIP. OK, thanks for clarifying.